lib/middleware/request_tracker.rb

```diff
@@ -166,7 +190,8 @@ def log_request_info(env, result, info, request = nil)
     data =
       begin
         self.class.get_data(env, result, info, request)
-      rescue StandardError
+      rescue StandardError => e
+        Rails.logger.warn("RequestTracker.get_data failed: #{e}")
```

Comment:
Not strictly relevant to this change... but I lost many hours of my life because of this silent error swallowing 😬